src/Internal/Testnet/Server.purs

```diff
@@ -281,129 +277,128 @@ startTestnetCluster cfg cleanupRef = do
       }
     pure { process: ogmios, channels: ogmiosChannels }
 
--- | Runs cardano-testnet executable with provided params.
-spawnCardanoTestnet
-  :: { cwd :: FilePath }
-  -> TestnetClusterConfig
-  -> Aff { testnet :: ManagedProcess, workspace :: FilePath }
-spawnCardanoTestnet { cwd } params = do
+-- | Spawns cardano-testnet process with provided parameters.
+spawnCardanoTestnet :: FilePath -> TestnetClusterConfig -> Aff ManagedProcess
+spawnCardanoTestnet workdir params = do
   env <- liftEffect Node.Process.getEnv
-  -- initCwd <- liftMaybe (error "Couldn't find INIT_CWD env variable")
-  --   $ Object.lookup "INIT_CWD" env
   let
     env' = Object.fromFoldable
-      [ "TMPDIR" /\ cwd -- only for 8.1.1; 8.7.2 puts it's testnet directory into cwd instead
-      -- , "CARDANO_NODE_SRC" /\ (initCwd <</>> "cardano-testnet-files")
-      , "CARDANO_CLI" /\ "cardano-cli"
-      , "CREATE_SCRIPT_CONTEXT" /\ "create-script-context"
+      [ "CARDANO_CLI" /\ "cardano-cli"
```

Comment:
Is it needed?

Reply:
Yes, without those vars, cardano-testnet will fail to run cardano-node and cardano-cli